integration-tests/package.json

```diff
@@ -34,9 +36,9 @@
     "human-id": "4.1.1",
     "ioredis": "5.4.1",
     "slonik": "30.4.4",
-    "strip-ansi": "7.1.0",
+    "strip-ansi": "6.0.1",
     "tslib": "2.8.1",
-    "vitest": "2.0.5",
+    "vitest": "2.1.8",
```

Comment:
Thought this might be the version that improves terminal watch mode flickering. Doesn't seem to have worked but no matter, upgrade vitest all the same.